scripts/migrations/v1-add-steady-type-down.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# uses jq, can be installed by npm i -g node-jq
+
+echo "creating backup users.backup.json"
+cp apps/backend/data/users.json apps/backend/data/users.backup.down.json
+
+jq 'del(.[].workouts[].parts[].type)' apps/backend/data/users.backup.down.json > apps/backend/data/users.json
```

Comment:
We should probably include it as a dependency, or we should run it through `npx`

```suggestion
npx node-jq 'del(.[].workouts[].parts[].type)' apps/backend/data/users.backup.down.json > apps/backend/data/users.json
```
I'm assuming this line will have some trouble with the `>` and so on 🥲

Reply:
and `npx` will ask if we want to install it, which requires some input from the user and so on.

It might be easiest to just include it as a dependency and then run it through npx